assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -224,12 +223,6 @@ const baseSelectors = {
 		}
 
 		if ( userPickedMetrics.length ) {
-			if ( ! isFeatureEnabled( 'conversionReporting' ) ) {
-				return userPickedMetrics.filter( ( slug ) => {
-					return keyMetricsGA4WidgetsNonACR.includes( slug );
-				} );
-			}
```

Comment:
This seems to be deleted by mistake, this is used in ACR epic, and is important to clear the metrics when feature flag is disabled, I will restore it in my current PR

Reply:
Here we've moved the filtering to within `getUserPickedMetrics` along with a new `displayInWidgetArea` check.

https://github.com/google/site-kit-wp/blob/f625d0286075066553879b4f26f667b20c032d78/assets/js/googlesitekit/datastore/user/key-metrics.js#L400-L425